<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>playlists_screen</class>
 <widget class="QWidget" name="playlists_screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>651</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="main_layout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMinimumSize</enum>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QWidget" name="account_w" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="songs_pb">
        <property name="text">
         <string>Songs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="tags_pb">
        <property name="text">
         <string>Tags</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="new_playlist_pb">
        <property name="text">
         <string>New Playlist</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="lists_w" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
